### ISSUE TITLE
Password for chat

### ISSUE THREAD

**Issue by readyyyk** [readyyyk/ChaPy#60]
Add password for chat.
Users should be able to protect their chats with password

**Comment by Ayush-Vish** [comment]
@readyyyk  I would like to work on this issue , Please assign it to me.

**Comment by readyyyk** [comment]
I don't think that you are enough familiar with this project architecture. This task requires both **server** functionality update and major client side update
